packages/analytics/shippers/elastic_v3/common/src/events_to_ndjson.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Event } from '@kbn/analytics-client';
+import { eventsToNDJSON } from './events_to_ndjson';
+
+describe('eventsToNDJSON', () => {
+  test('works with one event', () => {
+    const event: Event = {
+      timestamp: '2020-01-01T00:00:00.000Z',
+      event_type: 'event_type',
+      context: {},
+      properties: {},
+    };
+
+    expect(eventsToNDJSON([event])).toEqual(
```

Comment:
++ I initially implemented it with that. The resulting snapshot is:

```ts
    expect(eventsToNDJSON([event])).toMatchInlineSnapshot(`
      "{\\"timestamp\\":\\"2020-01-01T00:00:00.000Z\\",\\"event_type\\":\\"event_type\\",\\"context\\":{},\\"properties\\":{}}
      "
    `);
```

I switched to `toEqual` because it felt more explicit to make it clear that we are adding the `\n` at the end vs. realizing about the extra line before closing the double-quotes. Happy to switch change it though :)